docs/src/tool/index.md

```diff
@@ -292,55 +321,56 @@ development API key), and sets `REACH_RPC_TLS_REJECT_UNVERIFIED` to
 Consider this example from the @{seclink("tut-7-rpc")} tutorial:
 
 ```cmd
-$ reach rpc-run python3 -u ./index.py
+$ ./reach rpc-run python3 -u ./index.py
 ```
 
 # {#ref-usage-docker-reset} `reach docker-reset`
 
-You can easily kill and remove all Docker containers by executing
+This command removes all containers and volumes from Docker. If you have any personal containers, those containers will also be removed. Containers can then be restarted in your Docker Images list. It does not uninstall any Docker images.
```

Comment:
Is the "Docker Images list" referring to someplace in Docker Desktop's GUI? At a glance that might be confusing to people who live in the terminal and run everything from there.

Reply:
I agree.

Reply:
How about this? 

This command removes all containers and volumes from Docker. If you have any personal containers, those containers will also be removed. Containers can then be restarted . It does not remove any Docker images.

For someone like me who uses personal containers, I think it is important for them to know that 1. they will lose their container functionalities after doing this, and 2. this does not require everything to be reconfigured.